cumulusci/tasks/release_notes/tests/test_generator.py

```diff
@@ -122,6 +122,18 @@ def test_init_with_last_tag(self):
         self.assertEqual(generator.change_notes.current_tag, self.current_tag)
         self.assertEqual(generator.change_notes._last_tag, self.last_tag)
 
+    @responses.activate
+    def test_mark_down_link_to_pr(self):
+        pr = mock.Mock(number=1, html_url="http://pr", body="# Changes\r\n\r\nfoo")
+        github_info = self.github_info.copy()
+        self.mock_util.mock_get_repo()
+        generator = GithubReleaseNotesGenerator(
+            self.gh, github_info, PARSER_CONFIG, self.current_tag, self.last_tag
+        )
+        actual_link = generator._mark_down_link_to_pr(pr)
+        expected_link = "[[PR{}]({})]".format(pr.number, pr.html_url)
+        self.assertEquals(expected_link, actual_link)
```

Comment:
We should probably also add a test that is more of an integration test (but still using faked github data) showing that the generator will create the empty PRs section if there are PRs with no other change notes.

Reply:
You can see the missing test coverage here: https://coveralls.io/builds/25062620/source?filename=cumulusci%2Ftasks%2Frelease_notes%2Fgenerator.py#L150